### ISSUE TITLE
Copy Link input missing label

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-ALToolbox#346]
> On the "Share this website" screen, the text input containing the URL for users to copy does not have an associated label.

> Either add an accessible label to the input (using "aria-label" such as "Share link" or associating it with the visible "You can copy and share this link" text via "aria-labelledby"), or replace the readonly input with a non-form element like a "&lt;span&gt;" since the user does not type into this field. See WebAIM's guidance on form labeling: https://webaim.org/techniques/forms/controls

(See A11Y-09)

**Comment by BryceStevenWilley** [comment]
Moved this issue to the Toolbox; the issue is on the share page, but is mostly from `copy_button_html()`. It's full usage:

```python
copy_button_html(  interview_url(i=current_context().filename, style="short", new_session=1), label=al_copy_button_label.show(), tooltip_inert_text=al_copy_button_tooltip_inert_text.show(), tooltip_copied_text = al_copy_button_tooltip_copied_text.show() )
```

```html
<div class="al_copy">
<input readonly="" class="al_copy_value" type="text" value="http://localhost/start/ALAffidavitOfIndigency/affidavit/?from_list=1" aria-invalid="false">
<button type="button" aria-label="   class=" btn="" btn-secondary="" al_copy_button"="">
<span class="al_tooltip al_tooltip_inert" style="display: none;">Copy to clipboard</span>
<span class="al_tooltip al_tooltip_active" style="display: none;">Copied!</span>
<svg class="svg-inline--fa fa-copy" data-prefix="far" data-icon="copy" role="img" viewBox="0 0 448 512" aria-hidden="true" data-fa-i2svg=""><path fill="currentColor" d="..."></path></svg><!-- <i class="far fa-copy"></i> Font Awesome fontawesome.com -->
<span>Copy</span></button>
</div>
```

We should probably use the `text before` as the label for the input, or (better, IMO), make the input / text area a span / div, but styled the same way it is now.
